src/library/objects/Fleet.js

```diff
@@ -398,6 +398,7 @@ Contains summary information about a fleet and its 6 ships
 		switch(ConfigManager.elosFormula){
 			case 1: return this.eLos1();
 			case 2: return this.eLos2();
+			case 4: return this.eLos4();

```

Comment:
Did not set as default, please view and make adjustment if neededDid not set as default, please view and make adjustment if needed